scripts/apps/authoring/authoring/preview-custom-field.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {get, isEmpty} from 'lodash';
+
+import {getFields} from 'apps/fields';
+import {IArticle} from 'superdesk-interfaces/Article';
+import {IVocabulary} from 'superdesk-interfaces/Vocabulary';
+
+interface IProps {
+    item: IArticle;
+    field: IVocabulary;
+}
+
+export class PreviewCustomField extends React.PureComponent<IProps> {
+    render() {
+        const fields = getFields();
+        const {item, field} = this.props;
+        const FieldType = fields[field.custom_field_type];
+        const value = get(item.extra, field._id);
+
+        if (FieldType == null) {
+            console.warn('unkwnow custom type', field);
+            return null;
+        }
+
+        if (isEmpty(value)) {
```

Comment:
Isn't `!= null` enough?

Reply:
no idea what values might be stored there eventually, could be `{}`, `[]`, `''` and that still would probably mean no preview